Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -570,7 +570,7 @@ public IEnumerator ManipulationHandlerOneHandRotateWithConstraint()
 
         private class OriginOffsetTest
         {
-            const int numSteps = 1;
+            const int numSteps = 10;
```

Comment:
This made the test easier to see since i could see the hand animate. Note the test doesn't take much longer (~5 seconds instead of 1). I could move it back but I think many of our other tests animate hands and it makes it easier to see. Plus this emulated what real hands do better.